docs: added Edge troubleshooting steps (#1213)
spectrocloud/librarium: content/docs/14-troubleshooting/06-nodes.md

# Nodes & Clusters

> This page covers common debugging scenarios for nodes and clusters post-deployment.

Reviewer: ```suggestion
This page covers common debugging scenarios for nodes and clusters after they have been deployed.
```

Reviewer: Here, it's correct to exclude the hyphen because there's nothing after post deployment to modify, but the suggested change might be preferable. 

For example, a hyphen is correct in step 6 because 'read-only' modifies 'file system':

"Reboot the system to resume the default read-only file system."

# Nodes

## Scenario - Repaved Nodes

> Palette performs a rolling upgrade on the nodes when a change is detected in the `kubeadm` config. Below are some of the actions including but not limited to that will cause the `kubeadm` configuration to change and result in the nodes getting an upgrade:

Reviewer: ```suggestion
Palette performs a rolling upgrade on nodes when it detects a change in the `kubeadm` config. Below are some actions that cause the `kubeadm` configuration to change and result in nodes being upgraded:
```
